FlaxEditor/Surface/ContextMenu/VisjectCM.cs

```diff
@@ -314,39 +314,55 @@ public override bool OnKeyDown(Keys key)
         }
 
         /// <summary>
-        /// Gets the next visible sibling of a node
+        /// Gets the next siblings of a control.
         /// </summary>
-        private T GetNextVisible<T>(T item) where T : Control
+        /// <param name="item">A control that is attached to a parent</param>
+        /// <returns>An <see cref="IEnumerable{Control}"/> with the siblings that come after the current one.</returns>
+        private IEnumerable<Control> GetNextSiblings(Control item)
         {
-            if (item == null) return null;
+            if (item == null || item.Parent == null) yield break;
+
             var parent = item.Parent;
             for (int i = item.IndexInParent + 1; i < parent.ChildrenCount; i++)
             {
-                var child = parent.GetChild(i);
-                if (child.Visible && child is T nextItem)
-                {
-                    return nextItem;
-                }
+                yield return parent.GetChild(i);
             }
-            return null;
         }
 
         /// <summary>
-        /// Gets the previous visible sibling of a node
+        /// Gets the next siblings of a control that have a specific type.
+        /// </summary>
+        /// <param name="item">A control that is attached to a parent</param>
+        /// <returns>An <see cref="IEnumerable{T}"/> with the siblings that come after the current one.</returns>
+        private IEnumerable<T> GetNextSiblings<T>(Control item) where T : Control
+        {
+            return GetNextSiblings(item).OfType<T>();
+        }
+
+        /// <summary>
+        /// Gets the previous siblings of a control.
         /// </summary>
-        private T GetPreviousVisible<T>(T item) where T : Control
+        /// <param name="item">A control that is attached to a parent</param>
+        /// <returns>An <see cref="IEnumerable{Control}"/> with the siblings that come before the current one.</returns>
+        private IEnumerable<Control> GetPreviousSiblings(Control item)
         {
-            if (item == null) return null;
+            if (item == null || item.Parent == null) yield break;
+
             var parent = item.Parent;
             for (int i = item.IndexInParent - 1; i >= 0; i--)
             {
-                var child = parent.GetChild(i);
-                if (child.Visible && child is T prevItem)
-                {
-                    return prevItem;
-                }
+                yield return parent.GetChild(i);
             }
-            return null;
+        }
+
+        /// <summary>
+        /// Gets the previous sibling of a control that have a specific type.
+        /// </summary>
+        /// <param name="item">A control that is attached to a parent</param>
+        /// <returns>An <see cref="IEnumerable{T}"/> with the siblings that come before the current one.</returns>
+        private IEnumerable<T> GetPreviousSiblings<T>(Control item) where T : Control

```

Comment:
Would this (and the other similar methods in this file) be a good addition to Flax?